homework_10/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,53 @@
+import enums.LoggingLevel;
+
+public class FileLoggerConfiguration {
+    private String fileName;
+    private LoggingLevel loggingLevel;
+    private long fileSizeMax;
+    private String fileFormat;
+
+    public FileLoggerConfiguration(String fileName, LoggingLevel loggingLevel, long fileSizeMax, String fileFormat) {
+        this.fileName = fileName;
+        this.loggingLevel = loggingLevel;
+        this.fileSizeMax = fileSizeMax;
+        this.fileFormat = fileFormat;
+        if (fileName == null || fileFormat == null) {
```

Comment:
Why did you apply validation after assigning the field values? It should be done before assigning.